metadata/boundaries/county_boundaries.md

```diff
@@ -0,0 +1,74 @@
+# Title
+
+Utah County Boundaries
+
+## ID
+
+b2c8af56-3d92-4aa7-ac2f-b1388051d9ce
+
+## Brief Summary
+
+A statewide polygon dataset of county boundaries for cartography and approximate boundary identification.
+
+## Summary
+
+This dataset contains polygons for all 29 counties in Utah, where the perimeter of the polygon represents the county boundary. Each polygon also contains county population as of the [2020 Census](https://www.census.gov/library/stories/state-by-state/utah-population-change-between-census-decade.html) and current approximate population.
+
+## Description
+
+### What is the dataset
+
+This dataset represents each county as a polygon with information such as the Entity Number and [GNIS](https://www.usgs.gov/tools/geographic-names-information-system-gnis) code. This dataset also specifies what [state plane coordinate system](https://www.usgs.gov/faqs/what-state-plane-coordinate-system-can-gps-provide-coordinates-these-values#:~:text=The%20State%20Plane%20Coordinate%20System%20(SPCS)%2C%20which%20is%20only,the%20state's%20size%20and%20shape.) the county is in (North, Central, or South). Population data is derived from the 2020 Census (pop_lastcensus) and the Census Bureau's [Population Estimates Program](https://www.census.gov/programs-surveys/popest.html) (poplastest).
+
```

Comment:
We may want to add something like this to better explain the State Plane Coordinate System changes that will be coming in the near future. The State of Utah and NOAA’s National Geodetic Survey (NGS) is in the process of modernizing the National Spatial Reference System (NSRS).  As part of NSRS Modernization, the State Plane Coordinate System (SPCS) will be updated to SPCS2022. The new SPCS will greatly reduce distortion being projected to an average ground elevation unlike the older system that was projected to sea level. The proposed SPCS include one statewide zone, 11 multizone areas that will not follow County Boundaries similar to the 3 historic SP zones , and 79 Low Distortion Projections zones which can used in area with extreme elevation differences like roadways through canyons.  More information can be found here:
https://noaa.maps.arcgis.com/home/item.html?id=dddb7bc0be6f4e56a1c370c8d529d1a0

Reply:
Since these metadata are meant to be current and long lasting, I'd suggest that we be thoughtful and hesitant to add text describing the future. It creates technical debt since someone will need to set a reminder to make sure that the future statement is still accurate and update the text when the statement is no longer necessary. If you think it's worth the effort and will take care of those concerns then proceed. 

If you choose to proceed, i suggest adding text in the document that we can use with search to find the debt and validate the statements are still accurate.

Reply:
Since the state plane coordinate system is just one of the fields, and (as far as I understand it) not the focal point of the layer itself, I would say that we can just keep it vague and say that the feature class includes the attribute but not specify much else about it. It could just say "this dataset specifies what state plane coordinate system the feature is in" and that's all. That way, when the changes Sean mentioned are made, we won't need to update the metadata.